03-understanding-survey-data-files.Rmd

```diff
@@ -1 +1,238 @@
+
+Below, we show a question from the ANES 2020 questionnaire. This part shows a particular question's question name (`postvote_rvote`), description (Did R Vote?), full wording of the question and responses, response order, universe, question logic (if `vote_pre` = 0), and other specifications. It also includes the variable name, which can be used to link to the codebook.
+
+![](images/questionnaire-example.jpg)
+
+### Codebooks
+
+While a questionnaire provides information about the questions asked to respondents, the codebook explains how the survey data was coded and recorded. The codebook lists details such as variable names, variable label, variable meaning, codes for missing data, values labels, and value type (whether it is categorical or continuous, etc.). The codebook enables us to understand the variables and use them appropriately in our analysis.
+
+Below, we show a question from the ANES 2020 codebook. This part shows a particular variable's name (`V202066`), question wording, value labels, universe, and associated survey question (`postvote_rvote`).
+
+![](images/codebook-example.jpg)
+
+It is important to review both questionnaires and codebooks in parallel, as questions and variables are not one-to-one. A single question may have multiple associated variables, or a single variable may summarize multiple questions. Reviewing the codebook clarifies how to analyze the variables.
+
+### Errata
+
+An erratum (singular) or errata (plural) is a document that lists errors found in a publication or dataset, such as a survey questionnaire. The purpose of an erratum is to correct or update mistakes or inaccuracies in the original document.
+
+For example, if a survey questionnaire contains a question with an error, such as a typo or a confusing wording, an erratum would be created to provide a corrected version of the question. This helps to ensure the accuracy and reliability of the survey data and analysis.
+
+Be sure to review these lists before conducting any analysis to ensure that the most accurate and up-to-date information is used in the analysis.
+
+### Additional resources
+
+Surveys may have additional resources, such as interviewer instructions or "show cards" held up to respondents to help them answer a question. Review the survey website to find out more about these resources.
+
+## Working with missing data
+
+If a respondent does not reply to a survey question, we consider the data missing. Respondents may not respond to a question for a variety of reasons, such as not wanting to answer a particular question, not understanding the question, or simply forgetting to provide a response.
```

Comment:
Additionally, it may have been skipped by design. Mention this here but will be handled in next subsection